### PR TITLE
#DP-41 feat: pricing components

### PR DESCRIPTION
**Pull request by chanceDira**
# PR Description
This PR is for showing progress on pricing page allowing clients to access info about payment.

# Description of tasks that were expected to be completed
In this PR the expected tasks are, - designing cards, form and rendering components in pricing page

# Tasks
- [x] Pricing Card
- [x] Pricing form
- [x] render component in pricing pages

# How has this been tested?
Test the repo
```
git clone https://github.com/atlp-rwanda/atlp-pulse-fn.git
cd atlp-pulse-fn
git checkout ft-pricing-page-dp-41
npm install
npm run dev
```

(WIP) #dp-41

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -13,6 +13,21 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
+// create an app component
```
Comment (feyton) [inline src/index.tsx:16]: Despite the fact that we may need this code in the future, I see no reason to keep it while it is not currently being used.

You can safely remove it for now

File: package.json [hunk 2]
```diff
@@ -62,24 +68,24 @@
...
     "webpack": "^5.73.0",
     "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.9.2"
+    "webpack-dev-server": "^4.9.3"
   },
   "lint-staged": {
-		"*.{js, ts}": "eslint --fix",
-		"*.{js,ts,html,css,md}": "prettier --write  \"./**/*.{js,ts,jsx,tsx,json}\""
-	}
+    "*.{js, ts}": "eslint --fix",
```
[the remark this answers is not in the document]
Comment (chanceDira) [reply package.json]: Solved

File: src/index.tsx (hunk 1 above)
Comment (chanceDira) [reply src/index.tsx]: Solved.

**Review by samyshyaka — CHANGES_REQUESTED** [review]
Good job @chanceDira, Here below is some feedback

File: src/components/Header.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ const Header = () => {
   const handleClick = () => setNav(!nav);
 
   return (
-    <div className="w-screen h-[60px] z-10 bg-white dark:bg-dark-bg fixed border-b">
+    <div className="w-screen h-[60px] z-10 bg-white dark:bg-dark-bg border-b">
```
[the remark this answers is not in the document]
Comment (feyton) [reply src/components/Header.tsx:15]: The header should be fixed at the top

**Review by jeandamascyifuzo — APPROVED** [review]
Great job @chanceDira

**Comment by IIshimwe** [comment]
@chanceDira these cards can't look better if you minimize the list of things the chosen plan offers in order to reduce repeating in reading what a card offers you can add the word "Everything in basic plan" then short list of the things that aren't in the basic plan. That way I think you're getting the idea.

**Review by samyshyaka — APPROVED** [review]

**Review by feyton — APPROVED** [review]

**Pull request merged**